Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.yml

```diff
@@ -628,6 +628,24 @@ script:
     name: msgraph-user-change-password
     compliantpolicies:
     - User Soft Remediation
+  - name: msgraph-user-change-password-on-premise
+    description: |-
+      Changes the password of an on-premise user. Requires the UserAuthenticationMethod.Read.All permission. 
+      Providing a password is required (password auto-generation is not supported).
+    execution: true
+    arguments:
+    - description: User ID or userPrincipalName to update password for.
+      name: user
+      required: true
+      secret: false
+    - description: The new password.
+      name: password
+      required: false
```

Comment:
Why is it not required? According to the description, it should be

Reply:
You can use the non-sensitive arg as well. It is for playbooks